posthog-core/src/index.ts

```diff
@@ -116,6 +116,7 @@ export abstract class PostHogCore {
           {}
         )
       this.setKnownFeatureFlags(activeFlags)
+      options?.bootstrap.featureFlagPayloads && this.setKnownFeatureFlagPayloads(options?.bootstrap.featureFlagPayloads)
```

Comment:
we also need a way to _get_ all payloads, alongside `getAllFlags` in server-side libs.

Think it's worth combining those two together, keeps things simple, `getAllFlagsAndPayloads()`, which can then be directly passed to bootstrap.